<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>186</width>
        <height>180</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../samp2_4/AppIcon.icns</normaloff>../samp2_4/AppIcon.icns</iconset>
      </attribute>
      <attribute name="label">
       <string>QListWidget操作</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QToolButton" name="tBtnListInsert">
          <property name="text">
           <string>tBtnListInsert</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QToolButton" name="tBtnListAppend">
          <property name="text">
           <string>tBtnListAppend</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QToolButton" name="tBtnListDelete">
          <property name="text">
           <string>tBtnListDelete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="tBtnListClear">
          <property name="text">
           <string>tBtnListClear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="tBtnListIni">
          <property name="text">
           <string>tBtnListIni</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>186</width>
        <height>180</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QTreeWidget操作</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>186</width>
        <height>180</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QTableWidget操作</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>QListWidget</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>111</y>
         <width>451</width>
         <height>151</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>451</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>当前项变化:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="chkBoxListEditable">
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>451</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QToolButton" name="tBtnSelectItem">
          <property name="text">
           <string>tBtnSelectItem</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btnSelAll">
          <property name="text">
           <string>tBtnSelAll</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="btnSelNone">
          <property name="text">
           <string>tBtnSelNone</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="btnSelInvs">
          <property name="text">
           <string>tBtnSelInvs</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>listWidget</zorder>
      <zorder>gridLayoutWidget</zorder>
      <zorder>gridLayoutWidget_2</zorder>
      <zorder>listWidget</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QTreeWidget</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>QTableWidget</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actListIni">
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除项</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
